src/robot_dart/robot_dart_simu.hpp

```diff
@@ -118,14 +119,14 @@ namespace robot_dart {
 
         simu::GUIData* gui_data();
 
-        void enable_text_panel(bool enable = true);
+        void enable_text_panel(bool enable = true, float font_size = -1);
```

Comment:
In the robot_dart API everything is a double. We should do this as a double and cast it if necessary in other parts of the code.

Reply:
OK. Double is fine.